docs/components/camera/ffmpeg.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Configure a FFmpeg Camera"
+title: "Configure an ffmpeg Camera"
 linkTitle: "ffmpeg"
 weight: 30
 type: "docs"
-description: "Uses a camera, a video file, or a stream as a camera."
+description: "Uses a camera device, a video file, or a stream as a camera."
 images: ["/icons/components/camera.svg"]
 tags: ["camera", "components"]
 # SMEs: Bijan, vision team
 ---
 
-A `ffmpeg` camera uses a camera, a video file, or a stream as a camera.
+The `ffmpeg` camera model uses a camera device, a video file, or a stream as a camera, with support for streaming cameras that use MJPEG, H264, or MP4 tracks.
```

Comment:
ffmpeg supports much, much more than these types, too. you can do `ffmpeg -formats` to see the gigantic list it has

Reply:
Maybe say that it includes these formats, and more

Reply:
Updated, thanks!